spyderlib/qt/__init__.py

```diff
@@ -9,7 +9,7 @@
 
 import os
 
-os.environ.setdefault('QT_API', 'pyqt')
+os.environ.setdefault('QT_API', 'pyqt5')
```

Comment:
The commit introducing Qt5 compatibility is only 2 month old: 6629a92156eaaa29df58af6b28dd8dfb189bad45
Issues will pop when more people use it.

Adding it to the menu (in the dev version) could help to have more people testing it, however.